pom.xml

```diff
@@ -109,6 +109,7 @@
 		<maven.deploy.version>3.1.1</maven.deploy.version>
 		<spotbugs.version>4.7.3.5</spotbugs.version>
 		<jacoco.plugin.version>0.8.7</jacoco.plugin.version>
+		<jolokia.version>1.7.2</jolokia.version>
```

Comment:
Wondering whether this was too high up in the chain. I was hoping to have a way to keep versions consistent among third-party and internally in plugins. Then again, due to manifests etc. there is a question on to what extent managing versions centrally is a point at all. 

When it comes to the actual version. 1.7.2 has the needed features. I should probably retest with 2.0.1 and check to what extent the 2.0 stream works well enough. Is there a way to put PR in draft mode while we test?